web/src/components/index.css

```diff
@@ -365,6 +374,18 @@ body:not(.ios) .recording #background-container {
     }
 }
 
+@media (--mobile) {
```

Comment:
Those breakpoints are also old, the newer pages use more granular ones (4 different sizes). For more on CSS: https://github.com/mozilla/voice-web/issues/912

Reply:
Fixed as well.